content/concepts/consensus-network/amendments/known-amendments.ja.md

```diff
@@ -385,6 +388,16 @@ Checksトランザクションがアカウントのメタデータに影響を
 
 この変更により、受取人によるPayment Channelの検索が容易になります。また、アカウントがオープンPayment Channelの受取人だった場合に、そのアカウントが削除されないようにします（ただし、この修正の前に作成されたチャンネルを除きます）。
 
+
+## fixSTAmountCanonicalize
+[fixSTAmountCanonicalize]: #fixstamountcanonicalize
+
+| Amendment ID                                                     | ステータス    |
+|:-----------------------------------------------------------------|:----------|
+| 452F5906C46D46F407883344BFDD90E672B672C5E9943DB4891E3A34FEEEB9DB | 投票中 |
+
+Fixes an edge case in [deserializing](serialization.html) Amount-type fields. Without this amendment, in some rare cases the operation could result in otherwise valid serialized amounts overflowing during deserialization. With this amendment, the XRP Ledger detects error conditions more quickly to eliminate the problematic corner cases.
```

Comment:
`detects error conditions more quickly to eliminate the problematic corner cases`

This is really an "and" not a "to". We:
1. Detector error conditions more quickly and
2. Eliminate the problematic corner cases